.vsts/lint.yml

```diff
@@ -0,0 +1,17 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+steps:
+- bash: |
+    git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git "${AGENT_BUILDDIRECTORY}/depot_tools"
+    echo "##vso[task.setvariable variable=PATH]$PATH:${AGENT_BUILDDIRECTORY}/depot_tools"
+  name: Setup_depot_tools
```

Comment:
Can whitespace be used in the name? Like 'Setup depot_tools'.

Reply:
@alexeykuzmin no whitespaces don't work for some reason in VSTS

Reply:
@nornagon why do we need depot tools?

Reply:
@jkleinsc for cpplint & pylint?

Reply:
We should probably get rid of vendor/depot_tools.